src/main/java/edu/usf/cutr/OPC/FeedProcessor.java

```diff
@@ -138,10 +142,11 @@ else if (cause instanceof IndexOutOfBoundsException) {
                 }
                 
                 String dbTable = "[" +database + "].[dbo].[vehicle_positions]";
-                System.out.println("\nConnected to Database: " + dbInfo.getDatabase());
-		String sql = "SELECT TOP (15) [timestamp], [trip_id], [position_latitude], [position_longitude]\n" +
-                                "  FROM [gtfsrdb_HART_static_10-17-2016].[dbo].[vehicle_positions]" +
-                                "  WHERE [timestamp] >= ? AND [timestamp] <= ?";
+                System.out.println("\nConnected to Database: " + database);
+		String sql = "SELECT TOP(10000) [oid], [timestamp], [trip_id], [position_latitude], [position_longitude]\n" +
```

Comment:
Could you please remove the restriction on the `TOP(10000)` records here?  I'd like to be able to run the tool on all data that overlaps with the provided GTFS data (assuming we'll run this tool on non-HART datasets too).  I would suggest defaulting to running on all records, but also allow providing a command-line parameter to limit to the TOP X rows.